extension/api.js

```diff
@@ -0,0 +1,80 @@
+      let certDB = Cc["@mozilla.org/security/x509certdb;1"].getService(Ci.nsIX509CertDB);
+      // The last argument is a name which was actually ignored so later removed:
+      // https://dxr.mozilla.org/mozilla-esr52/rev/5350524b/security/manager/ssl/nsIX509CertDB.idl#418
+      certDB.addCertFromBase64(intermediate, ",,", "");
+      console.log("new intermediate certificate added"); // eslint-disable-line no-console
+    } catch (e) {
+      // If this fails, do not move on to the re-verify step.
+      return;
+    }
+
+    // Finally, force a re-verify of signatures.
+    try {
+      await XPIDatabase.verifySignatures();
+      Services.prefs.setBoolPref(PREF_HOTFIX_DONE, true);
+      console.log("signatures re-verified"); // eslint-disable-line no-console
+    } catch (e) {
+      console.error("failed to re-verify signatures:", e); // eslint-disable-line no-console
```

Comment:
nit: put `/* eslint-disable no console */` at the top of the file

Reply:
no-console*